R/dplyr.R

```diff
@@ -124,11 +124,13 @@ collect.tbl_BigQueryConnection <- function(x, ...,
   }
 
   quiet <- if (n < 100) TRUE else x$src$con@quiet
+  bigint <- if (!is.null(x$src$con@bigint)) x$src$con@bigint else "integer"
```

Comment:
```suggestion
  bigint <- x$src$con@bigint %||% "integer"
```

`%||%` is a handy helper from rlang